### DISCUSSION TITLE
Full featured desktop app

### DISCUSSION THREAD

**Discussion by POMXARK** [tauri-apps/tauri#5469] [Q&A]
Can I write a program to run on the client, that is, without unnecessary installations, everything should be standalone: database, interface. Please tell me the easiest way to implement this. Or will I have to choose another solution like с# wpf avalonia (reactive ui) + entity framework

**Comment by FabianLars** [comment]
we don't have fully portable app bundles (on windows) yet. But with a few drawback you can use the generated .exe without using the `.msi` installer:
- The msi installs WebView2 for you. So you'd have to provide that installation in other ways, either in Rust or via some external stuff (just list it as a requirement, or in case of enterprise applications there is often a way to distribute dependencies to all connected systems) - it's pre-installed on Windows 11 btw
- The `sidecar` and `resource` features only work with `.msi` bundles
- The Notification API needs the `.msi` to show the correct name/icon

Everything else that's not part of Tauri itself, like bundling databases is up to you.
